sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Properties/AssemblyInfo.cs

```diff
@@ -36,4 +36,4 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
```suggestion
[assembly: AssemblyVersion("0.10.0.0")]
```

Reply:
The general rule for preview module is: update assemblyversioon and assemblyfileversion to be the same as the version in csproj. Once it goes stable, update assemblyfileversion only unless it's a major version update.

Reply:
update to 
[assembly: AssemblyVersion("0.10.0.0")]
[assembly: AssemblyFileVersion("0.10.0.0")]